tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyAjax.java

```diff
@@ -1631,8 +1631,7 @@ public int compare(Map<String, Object> o1, Map<String, Object> o2) {
                     continue; // ignore unvotes.
                 }
                 PathHeader pathHeader = (PathHeader) m.get("pathHeader");
-                if (pathHeader.getSurveyToolStatus() != PathHeader.SurveyToolStatus.READ_WRITE &&
-                    pathHeader.getSurveyToolStatus() != PathHeader.SurveyToolStatus.LTR_ALWAYS) {
+                if (pathHeader.shouldSkipForVoting()) {
```

Comment:
This maybe should be `if (!pathHeader.canReadOrWrite())` instead -- currently the same result?  Study this further...

Reply:
I'm changing shouldSkipForVoting to !canReadOrWrite; no need for two functions.